src/c++/perf_analyzer/concurrency_worker.cc

```diff
@@ -40,6 +40,7 @@ namespace triton { namespace perfanalyzer {
 void
 ConcurrencyWorker::Infer()
 {
+  CreateCtxIdTracker();
```

Comment:
It could go in either place. Real code will always create a worker and call Infer on it once.

Currently we don't do anything in the constructor, so putting it here for consistency. Depending on how/where we put the factory this may change.